project/api/models/category.py

```diff
@@ -6,3 +6,9 @@ class Category(models.Model):
 
     created = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     last_modified = models.DateTimeField(auto_now=True, blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+    def __unicode__(self):
```

Comment:
Django 2.x and Python 3.x do not need the `__unicode__` method any longer, as strings are now unicode by default.